micro_dl/inference/image_inference.py

```diff
@@ -287,6 +309,7 @@ def _get_split_ids(self, data_split='test'):
         return split_col, inference_ids
 
     def _assign_3d_inference(self):
+        raise DeprecationWarning('3d inference no longer supported in 2.0.0')
```

Comment:
Since this is now deprecated, you can just delete all the 3D inference functions and tests and just keep the warning.